From 2852f5c3e712e41481ac4efdc4b12c5e5970b3fd Mon Sep 17 00:00:00 2001
From: steve ellcey-CA Eng-Software
 <sellcey@sellcey-thinkpad.caveonetworks.com>
Date: Fri, 14 Oct 2016 12:53:27 -0700
Subject: [PATCH] Fix warnings from latest GCC.

	* sysdeps/ieee754/dbl-64/e_pow.c (checkint) Make conditions explicitly
	boolean.
---
 sysdeps/ieee754/dbl-64/e_pow.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sysdeps/ieee754/dbl-64/e_pow.c b/sysdeps/ieee754/dbl-64/e_pow.c
index 663fa392c2..bd758b5979 100644
--- a/sysdeps/ieee754/dbl-64/e_pow.c
+++ b/sysdeps/ieee754/dbl-64/e_pow.c
@@ -466,15 +466,15 @@ checkint (double x)
     return (n & 1) ? -1 : 1;	/* odd or even */
   if (k > 20)
     {
-      if (n << (k - 20))
+      if (n << (k - 20) != 0)
 	return 0;		/* if not integer */
-      return (n << (k - 21)) ? -1 : 1;
+      return (n << (k - 21) != 0) ? -1 : 1;
     }
   if (n)
     return 0;			/*if  not integer */
   if (k == 20)
     return (m & 1) ? -1 : 1;
-  if (m << (k + 12))
+  if (m << (k + 12) != 0)
     return 0;
-  return (m << (k + 11)) ? -1 : 1;
+  return (m << (k + 11) != 0) ? -1 : 1;
 }
-- 
2.17.1

